perf/CMakeLists.txt

```diff
@@ -1,7 +1,20 @@
 set(CMAKE_CXX_STANDARD 14)
```

Comment:
Seems `perf/gh-7089-vclock-copy/` is not integrated to a build system and its results doesn't send to InfluxDB.
I propose to fix this.

Reply:
I've added the corresponding commit. I would ask @sergepetrenko to glance, because he is the author of the benchmark.